.github/workflows/ci-test.yaml

```diff
@@ -16,6 +16,9 @@ jobs:
       with:
         k8sVersion: ${{ matrix.k8sVersion }}
     - run: K8S_VERSION=${{ matrix.k8sVersion }} make ci-test
-    - uses: shogo82148/actions-goveralls@v1
-      with:
-        path-to-profile: coverage.out
+    - name: Install goveralls
+      run: go install github.com/mattn/goveralls@latest
```

Comment:
Can we install this as part of `.github/actions/install-deps`? Is there a good reason to do latest here rather than pinning a version?

Reply:
Same question, should this be part of our `toolchain.sh`?

Reply:
+1 on toolchain